client/src/components/Header.scss

```diff
@@ -1,9 +1,44 @@
 .Header {
+
+  .nav-wrapper {
+    background-color: rgb(95, 95, 95);
+    padding: 10px 10px 0 10px;
+    color: white;
+    height: 64px;
+    box-shadow:
+      0 2px 2px 0 rgba(0, 0, 0, 0.14), 
+      0 3px 1px -2px rgba(0, 0, 0, 0.12), 
+      0 1px 5px 0 rgba(0, 0, 0, 0.2);
+  }
+
   .nav-logo {
     display: inline-block;
+    width: -webkit-calc(100% - 320px);
```

Comment:
you can just do `calc(100% - 320px);` - the above only works for chrome/safari

Reply:
addressed along with a bunch of other stuff....let me know if there are any other changes you want me to make or if you want me to merge